### PR TITLE
Research Outputs Export

### PR DESCRIPTION
**Pull request by kmaid**
https://trello.com/c/v5T2OpQW/1257-5-export-search-results-to-csv

### CODE REVIEW

File: apps/frontend/src/shared-research/export.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    Promise<SearchResponse<V>>
+  >,
+  transform: (result: V) => Row,
+) => {
+  let morePages = true;
+  let currentPage = 0;
+  while (morePages) {
+    // We are doing this in chunks and streams to avoid blob/ram limits.
+    // eslint-disable-next-line no-await-in-loop
+    const data = await getResults({
```
Comment (ana-yldio) [inline apps/frontend/src/shared-research/export.ts:93]: how we handle when `getResults` fails?

**Review by peterstarling — APPROVED** [review]
it does seem to work although B stills needs to AC test it. Testing looks good too

File: apps/frontend/src/shared-research/export.ts (hunk 1 above)
Comment (kmaid) [reply apps/frontend/src/shared-research/export.ts:93]: Good catch! we don't and it silently fails. I am asking beatriz how we should handle this

Comment (kmaid) [reply apps/frontend/src/shared-research/export.ts:93]: image

**Pull request merged**